helm/portieris/templates/webhooks.yaml

```diff
@@ -1,12 +1,54 @@
+# webhook which is inoperative (no match in namespaceSelector) but in the helm manifest such that it it uninstalled on chart delete
 apiVersion: admissionregistration.k8s.io/v1beta1
 kind: MutatingWebhookConfiguration
 metadata:
   name: image-admission-config
+  annotations:
   {{ if .Values.UseCertManager }}
+    cert-manager.io/inject-ca-from: {{ .Release.Namespace }}/portieris-certs
+  {{ end }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ template "portieris.name" . }}
+    chart: {{ template "portieris.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+webhooks:
+  - name: trust.hooks.securityenforcement.admission.cloud.ibm.com
+    clientConfig:
+      service:
+        name: {{ template "portieris.name" . }}
+        namespace: {{ .Release.Namespace }}
+        path: "/admit"
+      {{ if not .Values.UseCertManager }}
+      caBundle: {{ .Files.Get "certs/ca.crt" | b64enc }}
+      {{ end }}
+    rules:
+      - operations: [ "CREATE", "UPDATE" ]
+        apiGroups: ["*"]
+        apiVersions: ["*"]
+        resources: ["pods", "deployments", "replicationcontrollers", "replicasets", "daemonsets", "statefulsets", "jobs", "cronjobs"]
```

Comment:
If this webhook is to be replaced by the other one, do we need the webhook rules list here or could it be empty? Or even could the webhooks list be empty - so that there's a MutatingWebhookConfiguration object but it contains no content?

Reply:
slimline webhook, slim enough?